test/k8s-integration/main.go

```diff
@@ -602,3 +597,32 @@ func deleteImage(stagingImage, stagingVersion string) error {
 	}
 	return nil
 }
+
+func getTempDir() string {
+	dir, err := ioutil.TempDir("", "")
+	if err != nil {
+		klog.Fatalf("Error creating temp dir: %v", err)
+	}
+	return dir
+}
+
+func removeTempDir(dir string) {
+	err := os.RemoveAll(dir)
+	if err != nil {
+		klog.Fatalf("Error removing temp dir: %v", err)
+	}
+}
+
+func ensureVariable(v *string, set bool, msgOnError string) {
```

Comment:
nit: if we're going to do more helper function for the flag handling we could break all of those out into a separate file

Reply:
I'll do this as part of the break things apart PR.